util/types/json/serdes.go

```diff
@@ -229,8 +229,10 @@ func encodeJSONString(s string, buffer *bytes.Buffer) {
 }
 
 func decodeJSONString(s *string, data []byte) (err error) {
+	var length uint64
 	var reader = bytes.NewReader(data)
-	if length, err := binary.ReadUvarint(reader); err == nil {
+	length, err = binary.ReadUvarint(reader)
+	if err == nil && length > 0 {
```

Comment:
If you parse `"\""\"""` into JSON as `j`, `j.s` will be an empty string with length = 0.